src/main/kotlin/no/nav/dagpenger/joark/mottak/JournalPostArkivHttpClient.kt

```diff
@@ -0,0 +1,52 @@
+package no.nav.dagpenger.joark.mottak
+
+import com.github.kittinunf.fuel.core.ResponseDeserializable
+import com.github.kittinunf.fuel.core.response
+import com.github.kittinunf.fuel.httpGet
+import com.github.kittinunf.result.Result
+import java.io.ByteArrayInputStream
+import java.io.InputStream
+import java.io.Reader
+import java.util.UUID
+
+class JournalPostArkivHttpClient(val joarkBaseUrl: String) : JournalpostArkiv {
```

Comment:
Since constructor arguments are default public `joarkBaseUrl` will be a public property (with setter and getter). It seems like this can be made private?

Reply:
Good point - will fix